example/main.go

```diff
@@ -15,8 +15,7 @@ import (
 
 	"github.com/hashicorp/raft"
 	"github.com/hashicorp/raft-boltdb"
-	"github.com/liftbridge-io/nats-on-a-log"
```

Comment:
Does the example compile without this import?

Reply:
no, it's need.

I check again my automatic code reformat script  and fix soon.